secret-tool: add page (#13728)
tldr-pages/tldr: pages/linux/secret-tool.md

# secret-tool

> Store and retrieve passwords, part of the `libsecret` package.

> > Communicates with `Secret Service` implementation such as `gnome-keyring`.

Reviewer: ```suggestion
> Communicates with Freedesktop secret service implementations such as `gnome-keyring`.
```

> > For more information <https://wiki.gnome.org/Projects/Libsecret>.

Reviewer: ```suggestion
> More information: <https://gnome.pages.gitlab.gnome.org/libsecret/>.
```
Since this GNOME Wiki page will be archived soon, I would suggest directly linking to the docs instead.

> - Store a secret with optional label:

Reviewer: ```suggestion
- Store a secret with an optional label:
```

`secret-tool store --label={{label}} {{key}} {{value}}`

- Retrieve a secret:

`secret-tool lookup key {{key}}`

> - Get more information about secret:

Reviewer: ```suggestion
- Get more information about a secret:
```
